### PR TITLE
Fix activity interfaces

### PR DESCRIPTION
**Pull request by roxblnfk**
## What does this PR do?

Fixes documentation typos

### CODE REVIEW

File: docs/dev-guide/phplang/foundations.md [hunk 1]
```diff
@@ -326,7 +326,7 @@ Returning results, returning errors, or throwing exceptions is fairly idiomatic
 However, Temporal APIs that must be used to get the result of a Workflow Execution will only ever receive one of either the result or the error.
 
 A Workflow method returns a Generator.
-To properly typecast the Workflow's return value in the client code, use the `#[ReturnType()]` annotation.
+To properly typecast the Workflow's return value in the client code, use the `#[ReturnType()]` attribute.
 
 ```php
 #[YourWorkflowInterface]
```
Comment (butschster) [inline docs/dev-guide/phplang/foundations.md:332]:
```suggestion
#[WorkflowInterface]
```

**Review by butschster — CHANGES_REQUESTED** [review]

**Comment by napcs** [comment]
@roxblnfk Thanks for the PR. I'm sorry for the late response.

It looks like you've made modifications to the generated file in the `docs` directory, rather than the source files. See the README.md file in the repository for contribution and build instructions. 

The short version is you should make changes to the appropriate files in the `docs-src` directory, and then run the local build steps to ensure things get built correctly. 

Let me know if I can help.

**Comment by roxblnfk** [comment]
Hi @napcs.
I followed [this guide](https://github.com/temporalio/documentation#how-to-fix-a-typo) and I got this error when trying to build the docs:
image
It would be nice if we could buld docs using a tested docker image.

**Comment by roxblnfk** [comment]
Reopened in #2379

**Pull request closed without merging**